djes/models.py

```diff
@@ -192,7 +203,16 @@ def mapping(self):
         return self.__class__.search_objects.mapping
 
     @classmethod
+    def is_orphaned(cls):
+        try:
+            return getattr(cls.search_objects.mapping.Meta, 'orphaned', False)
+        except:
```

Comment:
Shouldn't catch all exceptions -- what specific exception is expected that getattr()'s default value won't cover?